packages/clay-date-picker/src/Helpers.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Clone a date object.
+ * @param {!Date} d The date to clone
+ * @return {!Date} The cloned date
+ */
+export function clone(d) {
+	return new Date(d.getTime());
+}
+
+/**
+ * @param {!Date} d
+ * @return {!Date}
+ */
+export function getDaysInMonth(d) {
+	const firstDayOfMonth = new Date(d.getFullYear(), d.getMonth(), 1, 12);
+
+	firstDayOfMonth.setMonth(firstDayOfMonth.getMonth() + 1);
+	firstDayOfMonth.setDate(firstDayOfMonth.getDate() - 1);
+
+	return firstDayOfMonth.getDate();
+}
+
+/**
+ * Based on https://github.com/gpbl/react-day-picker/blob/master/src/Helpers.js#L55
```

Comment:
👍 great documentation!

Reply:
😅 I can not tell if this was good or bad haha. But I think I should better document what this function does...

Reply:
You might want to add a brief description of "how" its based on that code. If it was a straight copy of that code, its probably fine to have just the link.